src/DacpacTool/PropertyParser.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Globalization;
+using System.Linq;
+using System.Reflection;
+using Microsoft.SqlServer.Dac;
+
+namespace MSBuild.Sdk.SqlProj.DacpacTool
+{
+    /// <summary>
+    /// Allows to parse properties from build or deploy options
+    /// </summary>
+    public static class PropertyParser
+    {
+        private static readonly IDictionary<string, Func<string, object>> CustomParsers = new Dictionary<string, Func<string, object>>();
+
+        static PropertyParser()
+        {
+            CustomParsers.Add("DoNotDropObjectTypes", ParseObjectTypes);
+            CustomParsers.Add("ExcludeObjectTypes", ParseObjectTypes);
+            CustomParsers.Add("DatabaseSpecification", ParseDatabaseSpecification);
+        }
+
+        /// <summary>
+        /// Converts the <see cref="BuildOptions"/> to a <see cref="DacDeployOptions"/> object with all properties
+        /// </summary>
+        /// <param name="options">The build options</param>
+        /// <returns>The <see cref="DacDeployOptions"/> object</returns>
+        public static DacDeployOptions ToDacDeployOptions(this BuildOptions options)
```

Comment:
`ToDacDeployOptions` suggests we are converting one into the other, while actually we're just extracting a subset of the `BuildOptions` instance into a `DacDeployOptions` object. Perhaps `ExtractDeployOptions` would be a more appropriate name?

Reply:
I've renamed the method.